setup-environment

```diff
@@ -0,0 +1,9 @@
+#!/bin/bash
+
+python -m venv .venv
+source .venv/bin/activate
+pip install -e .
+pip install -r requirements.txt # we should merge these 2 together
```

Comment:
You can add `-e .` to `requirements.txt` to merge it.

Reply:
Since you included `-e .` I'd expect this to work and properly resolve all dependencies
```suggestion.
pip install -r requirements.txt
```